sympy/geometry/polygon.py

```diff
@@ -389,6 +455,49 @@ def centroid(self):
         return Point(simplify(A*cx), simplify(A*cy))
 
     @property
+    def directed_sides(self):
```

Comment:
Allright, so `evaluate` is not needed ? Will it be better if all sides are designated to be directed ?